pkg/Library/CoreXenon/Designer/GraphService.js

```diff
@@ -90,6 +96,13 @@ const createHostedLayer = async (layer, atom, graph, graphId, designable) => {
   }
 };
 
+const setLayerComposer = async (layer, Composer) => {
+  const onevent = (atomName, event) => App.handleAtomEvent(layer, atomName, event);
```

Comment:
I don't really understand what's happening here, maybe we can discuss it tomorrow (or whenever)